Resources/Prototypes/Language/languages.yml

```diff
@@ -576,3 +575,22 @@
     - hiss
     - ss
     - ee
+
+# Example of a sign language. Not currently used anyhow.
+- type: language
+  id: Sign
+  speech:
+    allowRadio: false
+    requireSpeech: false
+    color: "#dddddd"
+    messageWrapOverrides:
+      Speak: chat-sign-language-message-wrap
+      Whisper: chat-sign-language-whisper-wrap
+    speechVerbOverrides:
+    - chat-speech-verb-sign-1
+    - chat-speech-verb-sign-2
+    - chat-speech-verb-sign-3
+  obfuscation:
+    !type:ReplacementObfuscation
+    replacement:
+    - something
```

Comment:
```suggestion
    - something
    - a cryptic message to you
    - a signal to you
    - a message
    - # Blank - Does this work?
    - a rude expression to you
    - a sad expression to you
    - a happy expression to you
```

Reply:
The blank won't work, there are probably other ways to do it though

Reply:
Please just leave it simple, we dont need all of those.